lib/api_client.ex

```diff
@@ -12,13 +12,14 @@ defmodule GlassFactoryApi.ApiClient do
       {:ok , %{body: "[]", headers: [], status_code: 200}}
 
   If no configuration is passed, it will look for them in `Configuration.build\\1`
+  If no query string is passed, no filter is applied.
   """
 
-  @spec get(String.t(), map()) :: Tesla.Env.t()
-  def get(resource, configuration) do
+  @spec get(String.t(), [map()], map()) :: Tesla.Env.t()
+  def get(resource, query_string \\ [], configuration) do
```

Comment:
My only question is about this method, you are changing its arity from `/2` to `/3` it shouldn't break other places like [members](https://github.com/Cendrao/glass_factory_api/blob/master/lib/members.ex#L33) or [projects](https://github.com/Cendrao/glass_factory_api/blob/master/lib/projects.ex#L30)? 

Even `query_string` has an optional value when calling it like:

```elixir
ApiClient.get("members, config)
```

Looking at the function definition `def get(resource, query_string \\ [], configuration)`, wouldn't the config be the 2nd parameter and configuration come `nil` and we get an error?

Reply:
OBS: The tests are passing and I did a manual test in the console which worked too, I'm very confused about it 😅

Reply:
It's because of the default value (`query_string \\ []`). Every time I see this pattern I find it very confusing. Whenever possible, I'd leave the optional argument as the last one.

Reply:
I'm 🆗 with the query params in the middle, at least it is closer to the `resource`, which is more related to it than the config. But no strong opinions on it, feel free to swap them, the impact is small bc this is an "internal" API. About the name, I'd prefer `params` over `query_string`, because at this point it is not a string yet, it will only become a string when Tesla processes it.

Reply:
I don't have strong feelings to change too, I think we can leave this way.

I just was trying to understand 😄 Thanks @tegon!